src/NMFMerge.jl

```diff
@@ -8,12 +8,14 @@ export nmfmerge,
        mergecolumns
 
 """
-    result = nmfmerge(X, ncomponents; tol_final=1e-4, tol_intermediate=sqrt(tol_final), W0=nothing, H0=nothing, kwargs...)
+    result = nmfmerge(queuepenalty, X, ncomponents; tol_final=1e-4, tol_intermediate=sqrt(tol_final), W0=nothing, H0=nothing, kwargs...)
 
 Performs "NMF-Merge" on data matrix `X`.
 
 Arguments:
 
+-`queuepenalty`: a function of the form `f(E, t1sq, t2sq)` that computes the penalty for merging two components, where `E` is the the merge error described in the paper, default: f(E, t1sq, t2sq)=E.
+

```

Comment:
I added yesterday but forgot to mention in the conversation. I am confused about how to describe it in a good way.

Reply:
Maybe put the `queuepenalty` argument in square brackets, as that often indicates, an optional argument: `result = nmfmerge([queuepenalty,] X, ncomponents; ...)`. An alternative is to use multiple lines:
```julia
    result = nmfmerge(X, ncomponents; ...)
    result = nmfmerge(queuepenalty, X, ncomponents; ...)

Performs "NMF-Merge" on data matrix `X`. ...
```